src/make-http-request.coffee

```diff
@@ -25,7 +25,7 @@ makeHTTPRequest = (method, url, data, headers = {}, modify) ->
 
     req = req.set headers
 
-    if window?
+    if env is 'browser'
       req = req.withCredentials()
```

Comment:
The function doesn't exist in node, as browserify laods a different base class. https://github.com/visionmedia/superagent/blob/master/package.json#L50